src/components/teams/Teams.js

```diff
@@ -170,22 +164,65 @@ export default class Teams extends Component {
             </thead>
             <tbody>
              {teamsForRanking.map(teamRanking => {
-               return (
-                <tr key={teamRanking.clubId + teamRanking.teamCode}>
-                  <td>{teamRanking.position}</td>
-                  <td>{storeUtil.getClub(teamRanking.clubId).name + ' ' + teamRanking.teamCode}</td>
-                  <td>{teamRanking.gamesWon}</td>
-                  <td>{teamRanking.gamesLost}</td>
-                  <td>{teamRanking.gamesDraw}</td>
-                  <td>{teamRanking.points}</td>
-                </tr>
-                );
+               return this._getRanking(teamRanking);
              })}
             </tbody>
         </table>
     );
   }
 
+  _getTeamName(clubId) {
+    return storeUtil.getClub(clubId).name;
+  }
+
+  _getRankingAccordeon(teamRanking) {
+    var styling = {backgroundColor:'yellow',fontWeight:'bold'};
+    if (this._getTeamName(teamRanking.clubId) === 'Erembodegem') {
```

Comment:
werk met ID: ClubModel: `export const OwnClubId = 1;`
Als we van "Erembodegem" "TTC Erembodegem" maken, dan werkt deze code anders niet meer....